content/en/tracing/send_traces/_index.md

```diff
@@ -43,7 +43,11 @@ Note: After having instrumented your application, the tracing client sends trace
 
 ### AWS Lambda
 
-For more information setting up Lambda - X-Ray, see the [Amazon X-Ray integration documentation][7]
+To set up native tracing in AWS Lambda, see the [Lambda integration documentation][7]. For more information setting up Lambda - X-Ray, see the [Amazon X-Ray integration documentation][8].
```

Comment:
"Native tracing" is an internal term. We can just call this Datadog APM.

We can reword the second sentence to something like, "Alternatively, you can use AWS X-Ray to trace your Lambda functions."

Reply:
thanks, updated